rotkehlchen/tests/integration/test_blockchain.py

```diff
@@ -104,11 +106,19 @@ def mock_add_defi_balances_to_token_and_totals():
             blockchain=SupportedBlockchain.ETHEREUM,
             accounts=[addr1, addr2],
         )
+        ethtokens = EthTokens(database=blockchain.database, ethereum=blockchain.ethereum)
+        with ethtokens.db.user_write() as write_cursor:
+            ethtokens.detect_tokens(
+                write_cursor=write_cursor,
+                only_cache=False,
+                accounts=[addr1, addr2],
+            )
+
     assert addr1 in blockchain.accounts.eth
 
     with etherscan_patch, ethtokens_max_chunks_patch, defi_balances_mock, add_defi_mock, beaconchain_patch:  # noqa: E501
         greenlets = [
-            gevent.spawn_later(0.01 * x, blockchain.query_ethereum_balances, False)
+            gevent.spawn_later(0.01 * x, blockchain.query_ethereum_balances, ignore_cache=True)
```

Comment:
This test was useless before because cache was used instead of calling `query_ethereum_balances`. Spent quite some time figuring it out.

Reply:
It was not useless. This test was failing before. There was a bug where cache was not only ignored but something triggered an exception in the code.

With your change what are you achieving? You are just making it take a lot more time since you do 5 blockchain queries without any cache.

Reply:
It was not useless. This test was failing before, with the way the codebase was when this was written. There was a bug where cache was not only ignored but something triggered an exception in the code.

With your change what are you achieving? You are just making it take a lot more time since you do 5 blockchain queries without any cache.